doc/mgr/localpool.rst

```diff
@@ -3,37 +3,43 @@ Local Pool Module
 
 .. mgr_module:: localpool
 
-The *localpool* module can automatically create RADOS pools that are
-localized to a subset of the overall cluster.  For example, by default, it will
-create a pool for each distinct ``rack`` in the cluster.  This can be useful for
-deployments where it is desirable to distribute some data locally and other data
-globally across the cluster.  One use-case is measuring performance and testing
-behavior of specific drive, NIC, or chassis models in isolation.
+The ``localpool`` module can automatically create RADOS pools that are
+localized to a subset of the overall cluster.  For example, by default, it
+creates a pool for each distinct ``rack`` in the cluster. This can be
+useful for deployments where it is desirable to distribute some data locally
+and other data globally across the cluster. One use case is measuring
+performance and testing behavior of specific drive, NIC, or chassis models in
+isolation.
 
 Enabling
 --------
 
-The *localpool* module is enabled with::
+To enable the ``localpool`` module, run the following command:
 
-  ceph mgr module enable localpool
+.. prompt:: bash #
+
+   ceph mgr module enable localpool
 
 Configuring
 -----------
 
-The *localpool* module understands the following options:
+The ``localpool`` module understands the following options:
 
 .. confval:: subtree
 .. confval:: failure_domain
 .. confval:: pg_num
 .. confval:: num_rep
 .. confval:: min_size
 .. confval:: prefix
-   :default: by-$subtreetype-
 
-These options are set via the config-key interface.  For example, to
-change the replication level to 2x with only 64 PGs, ::
+The default is ``by-$subtreetype-``.
+
+These options are set via the ``config-key`` interface. For example, to
+change the replication level to 2x with only 64 PGs, run the following two commands:
```

Comment:
Not sure why we have "only" here, I would take it out.

Reply:
https://github.com/ceph/ceph/pull/63629